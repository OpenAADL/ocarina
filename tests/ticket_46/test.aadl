package p1 
public

d2 renames data d1;

private

data d1 end d1;

end p1;
