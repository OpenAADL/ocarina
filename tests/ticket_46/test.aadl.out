test.aadl:4:17: d1 (identifier) is not visible
Cannot analyze AADL specifications
